src/Microsoft.Management.Configuration/ConfigurationSetApplyProcessor.cpp

```diff
@@ -18,17 +18,30 @@ namespace winrt::Microsoft::Management::Configuration::implementation
             using namespace AppInstaller::Utility;
             return FoldCase(NormalizedString{ identifier });
         }
+
+        bool AssertFilter(ConfigurationUnitIntent intent)
+        {
+            return intent == ConfigurationUnitIntent::Assert;
+        }
+
+        bool InformFilter(ConfigurationUnitIntent intent)
+        {
+            return intent == ConfigurationUnitIntent::Inform;
+        }
+
+        bool ApplyFilter(ConfigurationUnitIntent intent)
+        {
+            return intent == ConfigurationUnitIntent::Apply || intent == ConfigurationUnitIntent::Unknown;

```

Comment:
Why does unknown implies apply? Is it because the group intent is unknown?

Reply:
The DSC v3 paradigm doesn't assign an intent at the unit level; groups are used to express the intent but we won't necessarily know what that intent is.  This is modeled with the `Unknown` intent, which is treated internally the same as `Apply`.